cpr/session.cpp

```diff
@@ -529,11 +508,19 @@ void Session::SetSslOptions(const SslOptions& options) {
     curl_easy_setopt(curl_->handle, CURLOPT_SSLVERSION,
                      // Ignore here since this has been defined by libcurl.
                      maxTlsVersion);
+
+    long curlSslOptions = 0;
```

Comment:
```suggestion
    // NOLINTNEXTLINE (google-runtime-int)
    long curlSslOptions = 0;
```

To make clang-tidy happy.